Makefile

```diff
@@ -305,7 +305,7 @@ release.controller:
 	@mkdir -p $(RELEASEDIR)
 	@tar -czf $(RELEASEDIR)/$(CONTROLLER_RELEASE_NAME).tar.gz --transform 's:^.*/::' $(BINDIR)/$(CONTROLLER_APP_NAME) README.md LICENSE
 
-release.sidecar.envoy:
+release.sidecar.envoy: build.sidecar
```

Comment:
not really. they should all be the same, this is just the one I was using. It can still be called explicitly in make, without setting the dependency. Will revert this line